routes/index.js

```diff
@@ -127,6 +129,52 @@ module.exports = function (passport) {
     });
   });
 
+  /*GET stationManagment route will render a table of all stations that match filter parameter.
+   *    If filter == "registered", only registered stations will be displayed (i.e. registered = true)
+   *    If filter == "unregistered", only unregistered stations will be displayed (i.e. registered = false)
+   *    Otherwise, all stations will be displayed sorted with unregistered stations first.
+   *********************************************************************************************************/
+  router.get('/stationManagement/:filter', jsonParser, function (req, res, next) {
+    //set filter based on query parameter passed in
+    var filter = req.params.filter;
+    filter = (filter === "registered") ? true : (filter === "unregistered") ? false : undefined;
+
+    //get results from database that match filter
+    dbAPI.getStations(filter).then(function(ret) {
+      //display ordered with unregistered stations first
+      /* UNCOMMENT THIS CODE WHEN TRAVIS HAS ACCESS TO A DATABASE SO RESULTS ARE SORTED
```

Comment:
Temporarily commented out code that dereferences the array of stations (to sort them). This would cause Travis CI tests to fail until it's able to establish a connection to a database. Until then, this getStations method will return null.